service/src/main/java/bio/terra/workspace/common/utils/AwsUtils.java

```diff
@@ -338,14 +338,24 @@ public static S3Client getS3Client(AwsCredentialsProvider awsCredentialsProvider
     return S3Client.builder().region(region).credentialsProvider(awsCredentialsProvider).build();
   }
 
```

Comment:
Nit: `@VisibleForTest` on `getSageMaker*()` ?

Reply:
already public, will me moved out to crl eventually